src/cluster_legacy.c

```diff
@@ -5314,6 +5318,11 @@ static int clusterNodeCronHandleReconnect(clusterNode *node, mstime_t now) {
     }
 
     if (node->link == NULL) {
+        if (now - node->outbound_link_attempt_time <= OUTBOUND_CONN_RETRY_INTERVAL || *cluster_conn_attempts > MAX_CONNECTION_ATTEMPTS_PER_CRON) {
```

Comment:
Would it make sense to also check node->inbound_link? If we have an inbound link, the node should be up, right?

```suggestion
        if (!node->inbound_link && (now - node->outbound_link_attempt_time <= OUTBOUND_CONN_RETRY_INTERVAL || *cluster_conn_attempts > MAX_CONNECTION_ATTEMPTS_PER_CRON)) {
```

Reply:
so if we have an inbound connection, we needn't require outbound to gossip. I think that makes sense.